src/systems/thruster/Thruster.cc

```diff
@@ -475,6 +514,19 @@ void ThrusterPrivateData::OnCmdThrust(const gz::msgs::Double &_msg)
   this->propellerAngVel = this->ThrustToAngularVec(this->thrust);
 }
 
+/////////////////////////////////////////////////
+void ThrusterPrivateData::OnDeabandEnable(const gz::msgs::Boolean &_msg)
+{
+  if (_msg.data() != this->enableDeadband)
+  {
+    if (_msg.data()) gzmsg << "Enabling deadband." << std::endl;
```

Comment:
Nit: For consistency, could you move to the next line the `gzmsg` instructions?

Reply:
I went ahead and ran `make codecheck` and fixed a few other code style issues.